src/main/java/edu/kis/powp/jobs2d/drivers/usageMonitor/DriverMonitor.java

```diff
@@ -0,0 +1,53 @@
+package edu.kis.powp.jobs2d.drivers.usageMonitor;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class DriverMonitor implements IDriverMonitor
```

Comment:
Please add info about a design pattern used in the name of the class. Now it is hard to understand the design from the name and hidden interfaces.

Reply:
Name changed to MonitorDriverDecorator.